dll.py

```diff
@@ -166,11 +268,12 @@ def __reversed__(self):
             yield node.value
             node = node.prev
 
-    def __contains__(self, item: T):
-        """Check if element is contained in the DoubleLinkedList
-        Now we can do `if elem in dll then <something> end`
-        :param item: Item of which we want to know if it is contained in the dll
-        :return: True if found, False otherwise
+    def __contains__(self, item: T) -> bool:
+        """
+        Returns whether a value is in the list
```

Comment:
Misschien wel handig om te melden hoe je dit normaal aanroept (want het is niet standaard om dll.__contains__ te doen)